src/resources/views/fields/latlng.blade.php

```diff
@@ -0,0 +1,204 @@
+
+        <script>
+        $(document).ready(function() {
+
+            // overwrite BackPack-CRUD submit on Enter
+            $(window).keydown(function(event){
+                if(event.keyCode == 13) {
+                    event.preventDefault();
+                    return false;
+                }
+            });
+
+            function initialize() {
+                var marker;
+                var default_zoom = {{$field['default_zoom']}};
+                var geolocate_icon = "{{$field['geolocate_icon']}}";
+                var latlong = document.getElementById('latlong').value; // get latlng value if any
+                latlong = latlong.split(',');
+                var latlng = (latlong == '') ? new google.maps.LatLng(3.138675,101.6167769) : new google.maps.LatLng(latlong[0], latlong[1]);
+                var geocoder = new google.maps.Geocoder();
+                var mapOptions = {
+                    center: latlng,
+                    zoom: 8,
+                    mapTypeId: google.maps.MapTypeId.ROADMAP,
+                    draggableCursor: "pointer",
+                    streetViewControl: false
+                };
+
+                // set search_address with formatted_address if latlng has value
+                (latlong == '') ? '' : setGeoCoder(latlng);
+
+                // set marker icon
+                var marker_icon = "{{$field['marker_icon']}}";
+                (marker_icon == '') ? null : marker_icon;
+                
+                var map = new google.maps.Map(document.getElementById('map-canvas'), mapOptions);
+            
+                var markers = new google.maps.Marker({
+                    position: latlng,
+                    map: map,
+                    icon : marker_icon
+                });
+            
+                google.maps.event.addListener(map, "click", function (location) {
+                    //map.setCenter(location.latLng); // set location to map's center each time map is clicked
+                    setLatLong(location.latLng.lat(), location.latLng.lng());
+
+                    marker.setMap(null);
+                    markers.setMap(null);
+                
+                    marker = new google.maps.Marker({
+                        position: location.latLng,
+                        map: map,
+                        icon : marker_icon
+                    });
+                    marker.setPosition(location.latLng);
+                    setGeoCoder(location.latLng);
+                });
+
+                var input = (document.getElementById('search_address'));
+
+                var autocomplete = new google.maps.places.Autocomplete(input);
+                autocomplete.bindTo('bounds', map);
+                autocomplete.setTypes([]);
+
+                var infowindow = new google.maps.InfoWindow();
+                marker = new google.maps.Marker({
+                    map: map,
+                    anchorPoint: new google.maps.Point(0, -29),
```

Comment:
Where'd that -29 come from (and the 0)?

Reply:
That is an offset of the infoWindow that will open from that marker. You can easily google that.

Reply:
I know what the values mean, I was more wondering why the particular values were chosen; sensible default I guess.